# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../cloop/kernel-only.diff
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- cloop-2.01/Makefile	2004-05-14 21:17:51.000000000 +0200
+++ cloop-2.01-fixed/Makefile	2004-10-16 15:58:19.406490400 +0200
@@ -32,7 +31,7 @@
 MODULE:=cloop.o
 endif
 
-ALL_TARGETS = create_compressed_fs extract_compressed_fs
+ALL_TARGETS = 
 ifndef APPSONLY
 ALL_TARGETS += $(MODULE)
 endif
@@ -73,6 +73,4 @@
 
 install:
 	mkdir -p $(MODDIR)
-	install -m 755 cloop.o $(MODDIR)
-	install -m 755 create_compressed_fs $(PREFIX)/sbin
-	install -m 755 extract_compressed_fs $(PREFIX)/sbin
+	install -m 755 $(MODULE) $(MODDIR)/
